### PR TITLE
Add example for org runner & update shutdown script

### PR DESCRIPTION
**Pull request by raj-saxena**
### Summary of changes:
1. Added example for organisation runners https://github.com/terraform-google-modules/terraform-google-github-actions-runners/issues/3
2. Noticed that there was a bug introduced with the organisation runner work that caused the organisation runner to not de-register properly. Updated the shutdown script to fix that.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_whitespace
 Failed whitespace check. More details below. 
 ```bash 
Checking for trailing whitespace
Checking for missing newline at end of file
Error: No newline at end of file ./modules/gh-runner-mig-vm/scripts/shutdown.sh
 ```

**Comment by ustuehler-brands4friends** [comment]
@googlebot I fixed it.

File: examples/gh-org-runner-mig-native-simple/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This example showcases how to use startup scripts to deploy organisation runners using the `gh-runner-mig` module.
+
+We use startup/shutdown scripts to install the runner binary, register the runner when it comes online and de-register when shut down.
+
+## Steps to deploy this example
+
+- Step 1: Create terraform.tfvars file with the necessary values.
+
+Access tokens require repo scope for private repos and public_repo scope for public repos. GitHub Apps must have the administration permission to use this API. Authenticated users must have admin access to the repository to use this API.
```
Comment (ustuehler-brands4friends) [inline examples/gh-org-runner-mig-native-simple/README.md:13]: I have yet to verify this, but I think the required token permissions should be described as follows:
```suggestion
Access tokens require the `organization_self_hosted_runners` permission to use this API.
```
Can you check which permissions are needed for the token, and update the documentation?

**Review by ustuehler-brands4friends — APPROVED** [review]
> Considering that this is an example for others to copy, and that it would be no longer internally consistent, could these hyphens be changed to underscores before merging?

I kind of regret saying that, since I noticed only afterwards that hyphens are used in a lot of places in this module. :( If the tests are still passing and @bharathkkb agrees though, then I'm happy with the new output variable naming in the examples.

File: examples/gh-org-runner-mig-native-simple/README.md (hunk 1 above)
Comment (raj-saxena) [reply examples/gh-org-runner-mig-native-simple/README.md]: Updated https://github.com/terraform-google-modules/terraform-google-github-actions-runners/pull/37/files#diff-fe9b11be3f708962a43495cfbc2fac9e34d23a180fcd8480a9a4682b3a08ac2eR13-R16

**Review by bharathkkb — APPROVED** [review]
Thanks for the PR @raj-saxena and comments @ustuehler-brands4friends. I agree with @ustuehler-brands4friends we should have standardized on underscores. We have an upcoming breaking release anyway so it's fine to fix these now :)

File: examples/gh-org-runner-mig-native-simple/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+output "mig_instance_template" {
+  description = "The name of the MIG Instance Template"
+  value       = module.runner_mig.mig_instance_template
+}
+
+output "gh_secret_id" {
+  description = "Secret Manager ID of the github secrets (token, repo_name,repo_owner)"
+  value       = module.runner_mig.gh_secret_id
+}
```
Comment (bharathkkb) [inline examples/gh-org-runner-mig-native-simple/outputs.tf:40]: @raj-saxena small nit, we can remove this. This is something the user supplies plus is  secret so we dont need to output this.

Comment (raj-saxena) [reply examples/gh-org-runner-mig-native-simple/outputs.tf]: Updated https://github.com/terraform-google-modules/terraform-google-github-actions-runners/pull/37/commits/4fff91f5fc3680ff60fb5fceaddc08fc24d16e1d

**Pull request merged**